examples/test/VisualizationByUri_test.js

```diff
@@ -82,3 +82,12 @@ test("Treemap should render", async t => {
 test("Heatmap should render", async t => {
     await checkRenderedChart(".s-visualization-heatmap", t);
 });
+
+test("Geo Pushpin chart should render", async t => {
```

Comment:
I prefer `GeoPushpinChart should render`.
What do you think?

Reply:
fixed